Resources/Textures/Objects/Specific/Chemistry/pills.rsi/meta.json

```diff
@@ -73,6 +73,12 @@
     {
       "name": "pill17"
     },
+    {
+      "name": "pill22"
+    },
+    {
+      "name": "pill23"
+    },
```

Comment:
these sprites just look like horizontally scaled round sprites

Reply:
yea i get that for the sprite 23 because it doesn't have a real world analogue, but sprite 22 will read as an estradiol pill to most people that know what they look like

i could probably drop the 23rd sprite?